lib/stack_master/stack_definition.rb

```diff
@@ -84,16 +84,25 @@ def s3_configured?
     def additional_parameter_lookup_file_paths
       return unless additional_parameter_lookup_dirs
       additional_parameter_lookup_dirs.map do |a|
-        File.join(base_dir, 'parameters', a, "#{underscored_stack_name}.yml")
+        [
+          File.join(base_dir, 'parameters', a, "#{underscored_stack_name}.yml"),
+          File.join(base_dir, 'parameters', a, "#{underscored_stack_name}.yaml"),
```

Comment:
Does this merge both contents, or does it prefer `yml`? (and should that be documented somewhere?)

Reply:
Good suggestion, it prefers `yml` now and it's documented and tested